classes/migration/upgrade/PKPv3_3_0UpgradeMigration.inc.php

```diff
@@ -15,6 +15,7 @@
 
 use APP\core\Application;
 use APP\core\Services;
+use APP\facades\Repo;
```

Comment:
This is a duplicate; it breaks the upgrade tests: https://app.travis-ci.com/github/pkp/ojs/builds/241578343

Reply:
Removed the `PKP\facades\Repo`;